solutions/APIO 11-path.mdx

```diff
@@ -2,13 +2,23 @@
 id: apio-11-path
 source: APIO 2011
 title: Find the Path
-author: Andi Qu
+author: Albert Ye, Andi Qu
 ---
 
-<IncompleteSection />
+# Solution
+The hardest part of this problem is designing the nodes and the edges to use in the Dijkstra. The framing of the problem means that it is most likely a Dijkstra problem.
```

Comment:
Could you add the complexity to the top of the code please?

Reply:
Also, I think you should swap these two sentences

Reply:
yea i'll do that